modules/lvms-limitations-to-configure-size-of-devices.adoc

```diff
@@ -23,36 +23,54 @@ ifdef::microshift[]
 endif::microshift[]
 
 ifndef::microshift[]
-.Size limits for different architectures using the default PE and LE size
-[cols="1,1,1,1,1", width="100%", options="header"]
+.Tested configuration
+[cols="1,1", width="100%", options="header"]
 |====
-|Architecture
-|RHEL 6
-|RHEL 7
-|RHEL 8
-|RHEL 9
 
-|32-bit
-|16 TB
-|-
-|-
-|-
+|Parameter
+|Value
 
-|64-bit
+|Chunk size
+|128 KiB
 
-|8 EB ^[1]^
+|Maximum volume size
+|32 TiB
 
-100 TB ^[2]^
-|8 EB ^[1]^
+|====
+
+.Theoretical size limits for static configuration
+[cols="1,1,1", width="100%", options="header"]
+|====
+
+|Parameter
+|Minimum value
+|Maximum value
+
+|Chunk size
+|64 KiB
+|1 GiB
+
+|Volume size
+|Minimum size of the underlying {op-system-first} system
+|Maximum size of the underlying RHCOS system
+
+|====
+
+.Theoretical size limits for host configuration
+[cols="1,1", width="100%", options="header"]
+|====
+
+|Parameter
```

Comment:
Usually, we would state the exact name of the parameters. These values seem like generalized descriptions. Does "Chunk size" have a direct code-based named such as "chunk_size"?

Reply:
Hi @jakobmoellerdev
Could you please help with this info?
Thanks! :)

Reply:
Hi @jakobmoellerdev 
The chunk size configuration is part of configuring a device? In that case I'm confused as to how this parameter is part of LVMCluster CR `.spec.storage.deviceClasses[].thinPoolConfig.chunkSize`.

Reply:
I just described the entire path, here is an example:

```yaml
apiVersion: lvm.topolvm.io/v1alpha1
kind: LVMCluster
metadata:
  name: my-lvmcluster
spec:
  storage:
    deviceClasses:
    - name: vg1
      default: true
      thinPoolConfig:
        name: thin-pool-1
        sizePercent: 90
        overprovisionRatio: 10
        chunkSize: 256Ki
        chunkSizeCalculationPolicy: Static
      deviceSelector:
        paths:
        - /dev/sda
      fstype: xfs
```

Reply:
Added the following new fields to the LVMCluster CR parameters table.
- chunkSize
- chunkSizeCalculationPolicy